### ISSUE TITLE
Namespace problem occured

### ISSUE THREAD

**Issue by capgemini-ocs** [svgdotjs/svg.js#465]
The library wrap the resulting SVG tag in another SVG tag at the end of the process. Then, it delete the extra SVG tag to return the result.

If the namespace is not HTML, the result is malformed : the root SVG tag is not removed properly because of the namespace. To fix this, the regular expression should be : 
`svg.js :: l.1113 : return well.innerHTML.replace(/^<svg( xmlns=".*?")?>/, '').replace(/<\/svg>$/, '')`
instead of :
`svg.js :: l.1113 : return well.innerHTML.replace(/^<svg>/, '').replace(/<\/svg>$/, '')`

**Comment by Fuzzyma** [comment]
I assume you are talking about the export function?
So [this](https://github.com/wout/svg.js/blob/595777968f56ebafc8ab0b25d8605ce258f87c98/dist/svg.js#L1216) line?

Furtermore I cant see a difference between
```javascript
return well.innerHTML.replace(/^/, '').replace(/<\/svg>$/, '')
// and
return well.innerHTML.replace(/^/, '').replace(/<\/svg>$/, '')
```

Anyway: If you want to create a PR for this dont hesistate to do that. It would help very much

**Comment by capgemini-ocs** [comment]
Ok, I just updated, it was a formatting issue. Now you can see the "xmlns" in the regexp.
And in fact, this is the line you gave.

**Comment by Fuzzyma** [comment]
Can you pls add a test for this to the testcase? This way we can assure, that the issue is fixed when we apply your patch. As I said: A Pull Request would be very welcome here

**Comment by Fuzzyma** [comment]
@capgemini-ocs I know people are short on time but it would be really helpful to have some code to test your issue. I dont like to introduce changes to the code without a real proof :)

**Comment by wout** [comment]
I'm closing this issue for now. Please let us know if you can make a more comprehensive proposal.

**Comment by Fuzzyma** [comment]
this was fixed in 4cf9fe96c6e03318a3748deb0338681360c57fdd in the [3.0 branch](https://github.com/svgdotjs/svg.js/tree/3.0.0)
